pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: pspdfkit_flutter
 description: PSPDFKit flutter plugin.
-version: 1.3.0
+version: 1.4.0
```

Comment:
Did we add any new features that justify a minor version bump? If not, a patch version bump seems more appropriate.

Reply:
We have adopted this behavior of bumping minor version whenever there's an update of PSPDFKit framework. Patch versions instead bump when there are minor changes that don't involve a new version of PSPDFKit version.

The reason behind is that updating to a new PSPDFKit framework can lead to (sometimes) many changes, this might have an impact that seems too big to justify just a patch version bump. This way a configuration like `1.4.+` can guarantee the same framework version.